### PR TITLE
Clarify README.md and make git-author script not print leading newlines

### PR DESCRIPTION
**Pull request by amilkh**

### CODE REVIEW

**Comment by amilkh** [comment]
Also, when soloing, an "Author:" is still added. Are we sure we want this?
```diff
 $ git log --pretty=full -1
 commit fef15b0319435ece09d40c9d3a810342c44c0cb2
 Author: Amil Khanzada <akhanzada@pivotal.io>
 Commit: Amil Khanzada <akhanzada@pivotal.io>
 
     centos-docker-setup: usability improvements + fix hwclock drift
+
+    Author: Amil Khanzada <akhanzada@pivotal.io>
```

**Comment by xinzweb** [comment]
Thanks a lot @amilkh. I tried to avoid having 'git-together' alias with 'git', since this will interfere other tools build on top of 'git'. To keep them separately, people can choose to use original 'git' behavior or 'git together' behavior.

Second, for solo author, I still prefer to have 'Author: ' there for following benefits:
- Help preserve authorship even the author field got updated by 'git together' or other 'git duet' tools during rebase or amend
- Easy for 3rd party tools to look for same pattern to look for author regardless how many people contributed to the commit.

Thanks,
Shin

File: git-author [hunk 1]
```diff
@@ -7,5 +7,5 @@ if [ -n "$1" ] ; then
 	git-together with $@ | sed "s/.*/Author: &/" >> $git_author_file_name
 fi
 
-cat $git_author_file_name
+git-together with
```
Comment (xinzweb) [inline git-author]: Hmm, the original intent is to verify we do set the template properly.

People can always manually call the `git-together with` to verify the `git together` is set properly.

My best practice would be always run `git author <new authors>` to ensure both template and the `git-together` are set properly.

**Comment by xinzweb** [comment]
+1 on the README change, but not the code change. Please split them, and I will merge the README part in. Thanks.

**Comment by amilkh** [comment]
> Thanks a lot @amilkh. I tried to avoid having 'git-together' alias with 'git', since this will interfere other tools build on top of 'git'. To keep them separately, people can choose to use original 'git' behavior or 'git together' behavior.

Discussed with David. Seems like we still need to do the alias for author rotation to happen automatically. What do you think?

File: git-author (hunk 1 above)
Comment (amilkh) [reply git-author]: Thanks good point.
David and I have updated the git-author script to use `awk` on the `~/.git_author` file instead to get the same output  behavior.

**Comment by amilkh** [comment]
@xinzweb any thoughts?

**Comment by xinzweb** [comment]
@amilkh the `git-together` alias is already done with the `git-together` repo setup, so I don't want to include in this setup. It's people's choice to alias it or not. The current setup will allow `git-author` with `git-together` regardless it's aliased as `git` or not.

Great patch, let's get it in.

**Comment by amilkh** [comment]
@xinzweb got it thanks!

**Pull request merged**